recipes/angsd/build.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+binaries="\
+misc/msToGlf \
+misc/contamination \
+misc/contamination2 \
+misc/splitgl \
+misc/thetaStat \
+misc/printIcounts \
+misc/NGSadmix \
+misc/realSFS \
+misc/smartCount \
+misc/supersim \
+angsd \
+"
+BINDIR=$PREFIX/bin
+mkdir -p $BINDIR
+
+make LDFLAGS="-L=${PREFIX}/lib" C_INCLUDE_PATH="-I=${PREFIX}/include" CPLUS_INCLUDE_PATH="-I=${PREFIX}/include"
```

Comment:
Most likely you only need either C_INCLUDE_PATH or CPLUS_INCLUDE_PATH. Further, you only specify the path in them, not the flag. Usually we define the env vars before running make, but it might work like this as well.